client/starwhale/api/_impl/wrapper.py

```diff
@@ -37,10 +37,11 @@ def close(self) -> None:
                 raise Exception(*exceptions)
 
     def _fetch_writer(self, table_name: str) -> data_store.TableWriter:
+        writer = self._writers.get(table_name)
```

Comment:
The tune code almost gains no performance benefit for python because of python GIL. And it brings the redundant code.